specification/components/stu3/schemas/responses/retrieveAdviceAndGuidanceWorklist/200Response.yaml

```diff
@@ -11,7 +11,7 @@ content:
     schema:
       $ref: '../../eRS-FetchWorklist-List.yaml'
     examples:
-      a&g-requests:
+      aAndg-requests:
         summary: Advice and guidance requests
```

Comment:
'&' cause linting error - this field is not displayed - so does not affect output